native-schema-registry/c/include/glue_schema_registry_schema.h

```diff
@@ -0,0 +1,41 @@
+#ifndef GLUE_SCHEMA_REGISTRY_SCHEMA_H
+#define GLUE_SCHEMA_REGISTRY_SCHEMA_H
+
+#include "glue_schema_registry_error.h"
+
+/*
+ * Glue Schema Registry Schema structure that represents
+ * schema object required by Glue Schema Registry Serializers / De-serializers.
+ */
+typedef struct glue_schema_registry_schema {
+    //String name of the schema
```

Comment:
don't use C++ comments in c header files if you can avoid it. Header files often need to be consumed by older language standards.

Reply:
Okay, I will update them.